spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require 'cog'
+require 'rspec/cog'
```

Comment:
It does feel a little sneaky to put this file in `rspec/cog` rather than `cog/rspec` but I guess that's what libs like `rspec-rails` do so maybe this is fine.